cssApp.css

```diff
@@ -49,31 +49,45 @@ html, body {
 	color: #555555;
 }
 
+.background {
+	display: flex;
+	justify-content: center;
+	background-size: 100% 100%;
+	padding: 100px 50px 0px;
```

Comment:
You don't need px with you have 0, so it would be:
padding: 100px 50px 0;

Reply:
Right, done